### ISSUE TITLE
Feature Suggestion: Flash Spectrum Notifications, perhaps with on demand line extractions

### ISSUE THREAD

**Issue by vnp85** [melix/astro4j#381]
**Context**
One of the key features of the spectral lines of the Sun is whether they show a strong _flash spectrum_. This is the realm of the chromosphere, prominences (hydrogen and calcium primarily, also with helium) and corona (iron primarily), ie stuff beyond the limb, and beyond what other, "boring" lines have to offer (say vanadium).

**Desired flow**
While recording, one encounters two kinds of flashes, beyond the limb of the Sun: 

- localized, belongs to the Sun, and 
- "full spectrum", appearing in all wavelengths, belongs to a ... dandelion or slit artefact. 
 
While analyzing the video, localized flashes and protrusions from the otherwise flat-ish limb could be reported into a list of pixel offsets from the main line. This would:
- autodetect, and perhaps auto-extract the hydrogen epsilon, when looking at the CaH line, without identifying the wavelength, from the BASS reference,
- aid the extraction of exotic lines that are hard to even locate, such as the higher hydrogen lines, helium lines, maybe iron lines at around 500 nanometers 

**Examples from my Observations**
When looking for interesting lines, I often look in the flash spectrum. I check either the hydrogen alpha/calcium locally, or do a quick search in gong hydrogen alpha or sdo's recent image. To find interesting lines to image, they need to be located, and for that, I need a relatively strong prominence. This is how I found the not-at-all obvious spectral lines, well beyond the *eye candy* the usual lines offer.

He I D3, next to an iron line and the sodium doublet, and calcium H with hydrogen eta, if not noted otherwise.

Thank you for considering.

![image](https://github.com/user-attachments/assets/662e5501-d789-47ed-acdc-0785912577c5)
![image](https://github.com/user-attachments/assets/f6c3aefc-542a-4c5b-8c76-365a18c16c44)
![image](https://github.com/user-attachments/assets/fceae426-2e98-4692-a4dc-c537f516749f)
![image](https://github.com/user-attachments/assets/d0e0582c-6307-446b-b081-3a932de573e7)
![image](https://github.com/user-attachments/assets/6c41687a-bdf2-4ee4-b8fd-22dcab9afc41)

**Comment by melix** [comment]
Yes, I have thought of doing something like that. It's not that simple, because it's an extension of detecting edges, which is already a hard problem (this is probably one of the areas I've most changed since the beginning of this project, and I'm still not happy with the result).

**Comment by melix** [comment]
@vnp85 I have something that prototypes this. It was capable of identifying the Helium D3 emission line automatically. However, I would need more data for validation. If you have SER files where you know these appear, would be happy if you could share links. Also ideally tell me which instrument and camera was used so that I can validate the wavelength computation. Thanks!
